### PR TITLE
Updating some dependencies to fix GH and npm security advisories (Closes #278)

### LINKED ISSUE #278: Scriptoni triggers security advisories
Having scriptoni as dependency triggers security advisories in other projects. See the following, for example:

image

Most of those come from `webpack-dev-server` cascade dependencies, and some dependencies of `smooth-release` and `metarpheus-io-ts` (e.g., lodash, axios, etc)

### PR DESCRIPTION
**Pull request by ecamellini**
Closes [#2521025](https://buildo.kaiten.io/space/[object Object]/card/2521025)

⚠️ The line above was added during the automatic migration of all github project issues to Kaiten. The old GH issue link is kept here for reference (it is now deprecated, continue follow the original issue on Kaiten).

Closes #278

### CODE REVIEW

**Comment by ecamellini** [comment]
Let's also wait for new releases of smooth-release and metarpheus-io-ts, after merges of https://github.com/buildo/smooth-release/issues/117 and https://github.com/buildo/metarpheus-io-ts/pull/111

**Review by giogonzo — APPROVED** [review]

**Pull request merged**
